### PR TITLE
Qml restructure - basic components part 1

### PR DESCRIPTION
**Pull request by gillesfernandez**
It includes the following components:
Button,
Back Button,
Text,
Text Field,
Disclaimer,
Footer

with Styling and a main.qml to test (./qmlscene main.qml)

### CODE REVIEW

File: internal/ui/qml-new/Controls/BackButton.qml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.1
+import "../Style"
+
+Item {
+    id: root
+
+    implicitHeight: 32
```
Comment (e8johan) [inline internal/ui/qml-new/Controls/BackButton.qml]: Should we add a DPI factor for sizes?

**Comment by e8johan** [comment]
Looks good to me. I mentioned the introduction of a DPI factor, but it can be added later if you want to.

File: internal/ui/qml-new/Controls/BackButton.qml (hunk 1 above)
Comment (gillesfernandez) [reply internal/ui/qml-new/Controls/BackButton.qml]: isn't it automatically fixed by Qt nowadays? As pixels don't represent points but are adapting depending on the display DPI?

**Comment by gillesfernandez** [comment]
Here is the last update:
- Login screen entirely recoded
- Cleaner split between the qml, go-app and go-server logics

I started cleaning up rpc.go, then stopped where I am now, after working with therecipe a little bit more, I see there is no easy way to do a "pure" Qt approach. I do though think rpc.go shall be split between the server management and basic communication, and the ui reactions to some actions. Something like a ServerProxy "class". I'd therefore would like to discuss with @holiman and @e8johan about what you'd think is best.

**Pull request merged**